src/topology/continuous_function/bounded.lean

```diff
@@ -238,6 +238,16 @@ begin
     exact λ N, (dist_le (b0 _)).2 (λx, fF_bdd x N) }
 end
 
+/-- Composition of a bounded continuous function and a continuous function. -/
+@[simps { fully_applied := ff }]
```

Comment:
Why did you add that `fully_applied := ff` argument? What happens if you don't put it here, that you want to avoid?

Reply:
I want `simp` to simplify `(f.comp_continuous g : δ → β)`, not only `f.comp_continuous g x`.